src/operator/account/AccountViewHeader.tsx

```diff
@@ -12,12 +12,14 @@ import {
   ComponentStatus,
 } from '@influxdata/clockface'
 import {Link} from 'react-router-dom'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
 // Contexts
 import {AccountContext} from 'src/operator/context/account'
 import {OperatorContext} from 'src/operator/context/operator'
 
 const AccountViewHeader: FC = () => {
+  const hideDeleteAccountButton = !isFlagEnabled('freeAccountCancellation')
```

Comment:
This feels backwards to me if I'm reading this right (and with the logic below). Should this be named `showDeleteAccountButton` instead?